### DISCUSSION TITLE
Comment syntax for API rules

### DISCUSSION THREAD

**Discussion by zakkor** [pocketbase/pocketbase#2609] [Q&A]
It would be nice to be able to add comments in API rules.
Some of these rules have gotten pretty complex, and I have no idea what they were supposed to do after returning back to them after a while.

<img width="197" alt="image" src="https://github.com/pocketbase/pocketbase/assets/6191421/42f4fc7f-331e-464e-9194-b83be9163fa7">

**Comment by ganigeorgiev** [comment]
Yes, comments are currently not supported in the rules syntax. I've added it to the roadmap to consider it for the next v0.18/v0.19 release.
